public/index.html

```diff
@@ -5,6 +5,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <link rel="apple-touch-icon" href="logo192.png" />
+    <link rel="stylesheet" href="index.css" />
```

Comment:
Don't we already have an `index.scss`?

Reply:
This is inside of public folder, as the style is applying to the index.html file in public.

Reply:
I guess my question is, why do we need a new file?

Reply:
I mean, if I load the actual site, I see both this include and the webpack "chunk", which means another XHR request, cache check, et cetera.  Does it not work if the rules are in `index.scss` and the styles are packed up with everything else?

Reply:
I think we should be able to remove the custom `index.css` and use the Boostrap `sr-only` class directly.